frontend/src/components/common/table/table/styles.module.scss

```diff
@@ -0,0 +1,54 @@
+:root {
+  --rectangle-5638-background: #242636;
+  --group-1000004448-color: #ffffff;
+  --rectangle-5639-color: #393b4d;
+  --group-1000004450-color: #6b6d7f;
+  --group-1000004448-font-family: roboto, sans-serif;
+  --group-1000004450-font-family: inter, sans-serif;
```

Comment:
add variables to `frontend/src/assets/css/variables.scss`

Reply:
you need to use the same names from figma 
<img width="245" alt="image" src="https://user-images.githubusercontent.com/42296182/183287177-892ed5f0-6263-49ce-8b4d-27f851436489.png">

Reply:
avoid changing font-family  for now